php/libraries/Database.class.inc

```diff
@@ -328,10 +329,15 @@ class Database
      * @param bool   $ignore     determines whether the insert throws and error or
      *                           is discarded when value exists in DB
      *
-     * @return void
+     * @return bool
```

Comment:
This function always return true. Is it required?

Reply:
Hmm I think we could add a `return false;` after line 371 for completeness. But I would still like to have the proper return type here. Before, the function would return void and true in different cases. At least now we can expect one type.

Reply:
Ok, keep it to true in that case.